backend/apps/membership/views.py

```
@@ -97,3 +99,37 @@ def destroy(self, request, *args, **kwargs):
                 {"detail": error_msg},
                 status=status.HTTP_400_BAD_REQUEST,
             )
+
+
+class MembershipAnalyticsViewSet(viewsets.ViewSet):
+    """
+    ViewSet for membership analytics.
+    """
+    permission_classes = [RequireScreenAccess('analytics')]
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)